### ISSUE TITLE
Documentation - issue with example required_providers block in README.md

### ISSUE THREAD

**Issue by danbarr** [cloudtamer-io/terraform-provider-cloudtamerio#1]
Using the example from the README.md, `terraform init` produces an error.

Terraform version 0.14.5

**Expected result:**
Successful `terraform init`

**Actual result:**
```
Error: Invalid provider type

  on main.tf line 3, in terraform:
   3:     cloudtamerio = {
   5:       source  = "github.com/cloudtamer-io/terraform-provider-cloudtamerio"
   6:     }

Provider source "github.com/cloudtamer-io/terraform-provider-cloudtamerio" has
a type with the prefix "terraform-provider-", which isn't valid. Although that
prefix is often used in the names of version control repositories for
Terraform providers, provider source strings should not include it.
```

**Fix:**
Replace `source  = "github.com/cloudtamer-io/terraform-provider-cloudtamerio"` with `source  = "cloudtamer-io/cloudtamerio"`

**Comment by danbarr** [comment]
You beat me to it - fixed in 089bb6400c99812b6a35752f9b11a0860ea1da76

**Comment by josephspurrier** [comment]
Awesome, thanks for finding this @danbarr!
